ROMFS/px4fmu_common/init.d/airframes/60002_uuv_bluerov2_heavy

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
+#
+# @name BlueROV2 (Heavy Configuration)
+#
+# @type Vectored 6 DOF UUV
+# @class Underwater Robot
+#
+# @output MAIN1 motor 1 CCW, bow starboard horizontal, , propeller CCW
+# @output MAIN2 motor 2 CCW, bow port horizontal, propeller CCW
+# @output MAIN3 motor 3 CCW, stern starboard horizontal, propeller CW
+# @output MAIN4 motor 4 CCW, stern port horizontal, propeller CW
+# @output MAIN5 motor 5 CCW, bow starboard vertical, propeller CCW
+# @output MAIN6 motor 6 CCW, bow port vertical, propeller CW
+# @output MAIN7 motor 7 CCW, stern starboard vertical, propeller CW
+# @output MAIN8 motor 8 CCW, stern port vertical, propeller CCW
+#
+# @maintainer Thies Lennart Alff <thies.lennart.alff@tuhh.de>
+#
+# @board px4_fmu-v2 exclude
+# @board intel_aerofc-v1 exclude
+# @board bitcraze_crazyflie exclude
+#
+
+. ${R}etc/init.d/rc.uuv_defaults
+
+if [ $AUTOCNF = yes ]
+then
+	#Set data link loss failsafe mode (0: disabled)
+	param set NAV_DLL_ACT 0
+
+	# disable circuit breaker for airspeed sensor
+	set CBRK_AIRSPD_CHK 162128
+	set CBRK_USB_CHK 197848
+	set CBRK_IO_SAFETY 22027
+	param set CBRK_AIRSPD_CHK ${CBRK_AIRSPD_CHK}
+	param set CBRK_USB_CHK ${CBRK_USB_CHK}
+	param set CBRK_IO_SAFETY ${CBRK_IO_SAFETY}
```

Comment:
Why is this not set directly? -> `param set CBRK_AIRSPD_CHK 162128`

Reply:
I guess i did it in the same manner the MAV_TYPE is set. I'll change as you suggested. What is the reason for setting MAV_TYPE via `set` **and** `param set` while the other values are set via `param set` only?

Reply:
Because `MAV_TYPE` is/was used in scripts as well. It could probably be cleaned up to avoid that.